src/components/modals/ReferenceModal.vue

```diff
@@ -2,12 +2,12 @@
   <Modal
     class="characters"
     @close="toggleModal('reference')"
-    v-if="modals.reference && roles.size"
+    v-show="modals.reference && roles.size"
```

Comment:
```suggestion
    v-if="modals.reference && roles.size"
```
See comment above.

Reply:
Please leave out remembering the maximize state for now and split that off into a separate PR. There's an easy way to do that on a per-modal level that's also pretty clean, but it would require rewriting every modal to get it working.
(it's basically how close() is handled in Modal.vue, so that maximize would emit an event that will be handled by the parent element, in this case the specific modal, and can thereby be saved per modal without much work)